frontend/components/OBSAdvAudioCtrl.cpp

```diff
@@ -25,7 +25,7 @@ OBSAdvAudioCtrl::OBSAdvAudioCtrl(QGridLayout *, obs_source_t *source_) : source(
 	QString sourceName = QT_UTF8(obs_source_get_name(source));
 	float vol = obs_source_get_volume(source);
 	uint32_t flags = obs_source_get_flags(source);
-	uint32_t mixers = obs_source_get_audio_mixers(source);
+	uint32_t mixersAssignmentsBitmap = obs_source_get_audio_mixers(source);
```

Comment:
No need to change the name. Technically it's a bitmask. If you insist on it, you could add a comment to explain that it is a bitmask storing the track selection as 1 << n etc. I think there's another place where I commented as much about that.